robyn/scaffold/prisma/app.py

```diff
@@ -0,0 +1,35 @@
+from robyn import Robyn
+from prisma import Prisma
+from prisma.models import User
+
+app = Robyn(__file__)
+prisma = Prisma(auto_register=True)
+
+
+@app.startup_handler
+async def startup_handler() -> None:
+    await prisma.connect()
+
+
+@app.shutdown_handler
+async def shutdown_handler() -> None:
+    if prisma.is_connected():
+        await prisma.disconnect()
+
+
+@app.get("/")
+async def h():
+    user = await User.prisma().create(
+        data={
+            "name": "Robert",
+        },
+    )
+    return user.json(indent=2)
+
+
+if __name__ == "__main__":
+    app.start()
+
+# install prisma cli
+# initialize prisma with "prisma generate"
+# then "prisma migrate dev" and name migration dev
```

Comment:
@bwdq 👋 

Can you please explain the line here?

Reply:
These are instructions to the user because prisma won't "just work" and requires some setup. I'm not sure what the optimal solution here is. I don't want to execute bash commands silently.

Reply:
hey @bwdq 👋 

We can add a small readme inside every folder. Doesn't need to be too fancy. Can just contain these commands.

Reply:
I have moved the python comments to README's.